pymc/distributions/timeseries.py

```diff
@@ -170,16 +172,16 @@ def rng_fn(
         ----------
         rng: np.random.RandomState
            Numpy random number generator
-        mu: array_like
+        mu: array_like of float
            Random walk mean
-        sigma: np.ndarray
+        sigma: array_like of float
             Standard deviation of innovation (sigma > 0)
-        init: float
+        init_dist: array_like of float
```

Comment:
float? Shouldn't this be a distribution?

Reply:
In the `rng_fn` we get non-symbolic inputs already

Reply:
Got it.